// @scalar/openapi-types/dist/schemas/extensions/index.js
import { XAdditionalPropertiesNameSchema } from "./x-additional-properties-name.js";
import { XCodeSamplesSchema, XCodeSampleSchema } from "./x-code-samples.js";
import { XEnumDescriptionsSchema } from "./x-enum-descriptions.js";
import { XInternalSchema } from "./x-internal.js";
import { XScalarIconSchema } from "./x-scalar-icon.js";
import { XScalarIgnoreSchema } from "./x-scalar-ignore.js";
import { XScalarRedirectUriSchema } from "./x-scalar-redirect-uri.js";
import { XScalarSdkInstallationSchema } from "./x-scalar-sdk-installation.js";
import { XScalarStabilitySchema, XScalarStabilityValues } from "./x-scalar-stability.js";
import { XTagGroupsSchema, XTagGroupSchema } from "./x-tag-groups.js";
import { XusePkceSchema, XUsePkceValues } from "./x-use-pkce.js";
export {
  XAdditionalPropertiesNameSchema,
  XCodeSampleSchema,
  XCodeSamplesSchema,
  XEnumDescriptionsSchema,
  XInternalSchema,
  XScalarIconSchema,
  XScalarIgnoreSchema,
  XScalarRedirectUriSchema,
  XScalarSdkInstallationSchema,
  XScalarStabilitySchema,
  XScalarStabilityValues,
  XTagGroupSchema,
  XTagGroupsSchema,
  XUsePkceValues,
  XusePkceSchema
};
